src/components/Form/Form.jsx

```diff
@@ -23,6 +24,12 @@ const Form = ({ formData, setFormData, answers }) => {
     console.log(formData);
   }, [score, name]);
 
+  // ########## SUGGESTION ##########
+  useEffect(() => {
+    // Here is something you need to do to solve your
+    // radio button bug.
+  }, [answers]);
```

Comment:
The bug is caused by the nature of uncontrolled input states of all HTML tags. The rule goes:
 "If not provided state, it tracks it on its own"
 
 For all `<input />` tags, there are different ways to control the state. The most common being
 `<input value={myValue]` />,. But when it comes to specifically radio inputs, they have a prop called "checked".
 
 Example:
 `<input checked={true} />`    // The radio button will always be checked.
`<input checked={false} />`    // The radio button will always be unchecked. 

When you change a tag from "uncontrolled" to "controlled", is when you control the state via a variable. In this case, you need to be able to control all radio buttons via their own boolean. And when you go to the next question, reset them all to false. As HTML wise, when you switch question, its still the same input tag, so its own internal state is preserved.